.vsts-pr.yml

```diff
@@ -100,8 +100,8 @@ stages:
     - template: eng/build-pr.yml
       parameters:
         agentOs: Linux
-        jobName: Build_Debian_Bullseye_Debug_x64
-        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11'
+        jobName: Build_Debian_11_Debug_x64
+        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-amd64'
```

Comment:
I see a combination of quotes and not. Is there a preference for this?

Reply:
I am not aware of one.  IMO there is no need for the quotes here.  But I also don't want to start including those types of changes in this PR.

Reply:
In YAML/JSON, quotes let it explicitly know it is a string type. They're only necessary if the type could be questionably be interpreted as another type, like if you started the string with numbers. Here, it is definitely a string, so you don't need the quotes.